cmake/modules/FindGLEW.cmake

```diff
@@ -62,6 +62,12 @@ if (WIN32)
         PATH_SUFFIXES
             Release/${ARCH}
         DOC "The GLEW library")
+
+    if ("${GLEW_LIBRARY}" MATCHES "glew32s(d|)")
+        add_definitions("/DGLEW_STATIC")
```

Comment:
Yeah, that's what I thought from the docs too, however Windows has a quirk where global variables declared "extern" will not be shared over library boundaries unless they are declared with dllexport declspec. Looking at the GLEW_STATIC/GLEW_BUILD preprocessor logic, the way to get dllexport is by declaring GLEW_BUILD.

So it seems this is the only way to make it work, unless you are ok with unique glew state per-library, but the USD imaging init pattern requires shared state, since Glf calls glewInit() (via GlfGlewInit); so without shared state, Glf will be the only library that gets initialized correctly.